sm/engine/dataset_manager.py

```diff
@@ -170,6 +175,7 @@ def __init__(self, db, es, image_store, mode, action_queue=None, status_queue=No
         self._action_queue = action_queue
 
     def _post_sm_msg(self, ds, action, priority=DatasetActionPriority.DEFAULT, **kwargs):
+        ds.set_status(self._db, self._es, self._status_queue, DatasetStatus.QUEUED)
```

Comment:
This `set_status` is duplicated in the below `if` block

Reply:
thanks, fixed